Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,32 @@
+
+## A pdf reorder or Rearranging Script
+
+#### I used the pdfrw library 
```

Comment:
Don't use headings unnecessarily.

Reply:
Okay will go over the existing readme files and will create a better one